lib/prx_auth/rails/ext/controller.rb

```diff
@@ -1,8 +1,14 @@
+require 'prx_auth/rails/token'
+
 module PrxAuth
   module Rails
     module Controller
       def prx_auth_token
-        request.env['prx.auth']
```

Comment:
I'm curious if this will break anything.  I want to say there are some CMS tests directly messing with this?

Reply:
I only found one such test, and I will fix that when I pull this in to stub the method instead of the rack value.